index.html

```diff
@@ -0,0 +1,110 @@
+            <li> <img src="http://i.imgur.com/vCWEjwG.png"> </li>
+          </a>
+          <a href="">
+            <li> <img src="http://i.imgur.com/wxiFpvM.png"> </li>
+          </a>
+          <a href="">
+            <li> <img src="http://i.imgur.com/jrfNOkO.png"> </li>
+          </a>
+        <ul>
+      </nav>
+    </header>
+    <main>
+      <article>
+        <header class="recipe-header">
+          <h1> Chocolate Pizza </h1>
+          <p> POSTED ON <time datetime="2013-12-15"> 15 DEC 2013 </time> / DESSERTS </p>
+          <section class="print">
+            <img src="http://i.imgur.com/mtX28lM.png">
+            <p> PRINT </p>
+          </section>
+          <img class="food-pic" src="http://i.imgur.com/fB4svJW.png">
+        </header>
+        <section class="recipe-instructions">
+          <p> For the fig-swirl: Melt butter over medium heat in a saucepan. Add brown sugar and stir to dissolve. Halve all of the figs and toss in the saucepan with water and lemon juice. Cook over medium heat, stirring frequently, until you have a chunky-jammy mixture. Add salt with one or two stires, set aside and let cool completely. </p>
+          <p> Ice cream: In a small pot over medium head, combine milk, and granulated sugar until sugar is completely dissolved and the milk is just barely lukewarm. Whisk in the egg yolks. Set mixture in the fridge and wait until the fig mixture is cooled. </p>
+          <p> Using an ice cream machine, pour liquids into the frozen basin and process according to the manufacturer instructions, i.e., let spin and thicken for 20 minutes before adding mascarpone, fig jam mixture, and the nuts. Continue to process for +/- 10 minutes. Pour semi-frozen mixture into a pyrex dish or glass tupperware. Freeze for at least two hours before serving. </p>
+        </section>
+        <section class="recipe-ingredients">
+          <ul>
+            <li> 1 1/2 cups milk </li>
+            <li> 1/2 cup mascarpone </li>
+            <li> 1/2 tsp pink salt </li>
+            <li> 1 lb Black Mission Figs </li>
+            <li> 1/2 cup brown sugar </li>
+            <li> 2-4 tbsp water </li>
+            <li> 1 /12 cups heavy cream </li>
+            <li> 1/3 granulated sugar </li>
+            <li> 2 egg yolks </li>
+            <li> 1 lemon, juiced </li>
+            <li> 2 tbsp butter </li>
+            <li> 1 cup honey roasted pecans, roughly chopped </li>
+          </ul>
+        </section>
+        <section class="recipe-author">
+          <img src="http://i.imgur.com/fTLnx6g.png">
+          <h2> Vanessa Stevenson </h2>
+          <p> Food enthusiast, photography fan. Add a pinch of raw foodism and that's pretty much who I am. </p>
+        </section>
+      </article>
+    </main>
+    <footer>
+      <img src="http://i.imgur.com/AiBJ4vH.png">
+      <p> Delicious &copy 2013 - All Rights Reserved. <br> Proudly published with Ghost. </p>
+      <button name="share" type="button"> SHARE RECIPE </button>
```

Comment:
Avoid the use of 'button' elements outside of 'form' elements. It's more semantic to style a link with a class "button" or "button-like" if it's simply a link.